src/gfn/utils/prob_calculations.py

```diff
@@ -103,23 +104,34 @@ def get_trajectory_pfs(
             valid_actions.tensor
         )  # Using the actions sampled off-policy.
 
-        log_pf_trajectories = torch.full_like(
-            trajectories.actions.tensor[..., 0],
```

Comment:
what's the rationale behind removing this?

Reply:
This was just moved to line 78 to address the edge case in line 84!